docs/guides/rtc_product_guide.md

```diff
@@ -46,7 +46,7 @@ The RTC product package includes a Layover-Shadow mask (see [Image Files section
 
 ## Digital Elevation Models
 
-The quality of the terrain corrections are directly related to the quality of the digital elevation models (DEMs) used in the process of geometrically and radiometrically correcting the SAR imagery. We use DEMs that are publicly available and have wide-ranging coverage. 
+The quality of the terrain corrections are directly related to the quality of the digital elevation models (DEMs) used in the process of geometrically and radiometrically correcting the SAR imagery. Cell values of the DEMs indicate the elevation in meters. We use DEMs that are publicly available and have wide-ranging coverage. 
```

Comment:
I think this might fit better in the paragraph describing Table 1, where we discuss the projection and geoid correction.